pensjon-brevbaker-api-model/src/main/kotlin/no/nav/pensjon/brev/api/model/maler/redigerbar/InnvilgelseAvAlderspensjonDto.kt

```diff
@@ -0,0 +1,98 @@
+package no.nav.pensjon.brev.api.model.maler.redigerbar
+
+import no.nav.pensjon.brev.api.model.AlderspensjonRegelverkType
+import no.nav.pensjon.brev.api.model.Sakstype
+import no.nav.pensjon.brev.api.model.maler.BrevbakerBrevdata
+import no.nav.pensjon.brev.api.model.maler.RedigerbarBrevdata
+import no.nav.pensjon.brev.api.model.vedlegg.DineRettigheterOgMulighetTilAaKlageDto
+import no.nav.pensjon.brev.api.model.vedlegg.MaanedligPensjonFoerSkattAP2025Dto
+import no.nav.pensjon.brev.api.model.vedlegg.MaanedligPensjonFoerSkattDto
+import no.nav.pensjon.brevbaker.api.model.DisplayText
+import no.nav.pensjon.brevbaker.api.model.Foedselsnummer
+import no.nav.pensjon.brevbaker.api.model.Kroner
+import java.time.LocalDate
+
+
+@Suppress("unused")
+data class InnvilgelseAvAlderspensjonDto(
+    override val pesysData: PesysData,
+    override val saksbehandlerValg: SaksbehandlerValg,
+) : RedigerbarBrevdata<InnvilgelseAvAlderspensjonDto.SaksbehandlerValg, InnvilgelseAvAlderspensjonDto.PesysData> {
+
+    data class SaksbehandlerValg(
+        @DisplayText("Virkningstidspunktet er senere enn ønsket uttakstidspunkt")
+        val kravVirkDatoFomSenereEnnOensketUttakstidspunkt: Boolean,
+        @DisplayText("Gjenlevenderett er brukt i beregningen")
+        val harGjenlevenderett: Boolean,
+        @DisplayText("Gjenlevendetillegg er større enn 0")
+        val harGjenlevendetillegg: Boolean,
+        @DisplayText("GjenlevendetilleggKap19 kommer til utbetaling")
+        val harGjenlevendetilleggKap19: Boolean,
+        @DisplayText("Hvis egen opptjening er best")
+        val egenOpptjening: Boolean,
+        @DisplayText("Hvis supplerende stønad")
+        val supplerendeStoenad: Boolean,
+        @DisplayText("Hvis kildeskatt")
+        val kildeskatt: Boolean,
+        @DisplayText("Hvis ikke kildeskatt")
```

Comment:
Det skurrar litt for meg å ha både `kildeskatt` og `ikkeKildeskatt`, kan desse vera ein og samme boolean, og så sjekkar vi heller med `.not()` på den?

Reply:
Har fjernet val for "ikke kildeskatt"